product/common-src/i18n/localeDataEn.ts

```diff
@@ -59,6 +59,14 @@ export const localeDataEn: ILocaleData = {
   MENU_helpOnline: 'Online Help',
   MENU_helpQuestion: 'Report Issues',
 
+  HTML_lang: 'en',
+  HTML_availableFirefoxSafariChrome: 'Animation available on Firefox, Safari and Chrome',
+  HTML_cantAnimateOnIE: 'IE does not support to play the animation',
+  HTML_cantViewOnIE: 'IE does not support to display the image',
+  HTML_forWebpSupportedBrowsers: 'for webp supported browsers',
+  HTML_forWebpUnsupportedBrowsers: 'for webp unsupported browsers',
```

Comment:
```suggestion
  HTML_forWebpSupportedBrowsers: 'for WebP supported browsers',
  HTML_forWebpUnsupportedBrowsers: 'for WebP unsupported browsers',
```
拡張子としてみたら正しいので悩ましいところなんですが、ほかで使っている名称のところでAPNGやWebPと大文字にする形になっているので、こちらも合わせてあげたいです。